Arrowgene.Ddon.Shared/Files/Assets/quests/q20000002.json

```diff
@@ -61,8 +65,16 @@
         {
           "type": "MyQstFlags",
           "announce_type": "Accept",
-          "set_flags": [ 6 ],
-          "check_flags": [ 1, 2, 3, 4, 5 ]
+          "set_flags": [
```

Comment:
These JSON formatting changes made to all these quests makes it harder to read which is important since we are hand writing all these quests.